CMakeLists.txt

```diff
@@ -1293,7 +1306,7 @@ install(TARGETS blasfeo EXPORT blasfeoConfig
 	RUNTIME DESTINATION bin)
 
 
-install(EXPORT blasfeoConfig DESTINATION cmake)
+install(EXPORT blasfeoConfig DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/blasfeo FILE blasfeoConfig.cmake)
```

Comment:
I just checked and this line makes my default acados installation fail, as it tries to write to `/cmake/blasfeo`.
A solution could be adding a CMake variable `BLASFEO_CMAKE_INSTALL_DIR` and implement the default as it is.
But I am not sure what the actual issue is that motivated this change.

Reply:
In a Yocto environment the Config.cmake couldn’t be found because the install location didn’t follow the GNUInstallDirs policy. Yocto (like most distros) relies on the GNU layout, typically /usr/share/cmake/<pkg>, so installing there fixes detection.

Introducing the new variable BLASFEO_CMAKE_INSTALL_DIR would not be 100% compatible with previous workflows, since they would need to configure this variable now. Why adjust for the GNU layout instead?

Reply:
Sounds good! I just tested with acados and it works without sudo. I will just gitignore the share folder in acados then.
From my side this can be merged then.